### PR TITLE
New sink sprites + rectangle mirrors

### PR DESCRIPTION
**Pull request by nicbn**
image
image
image

Also mirrors are now rectangular.

## Changelog
:cl:
soundadd: New sink sprites. Hooray? Also mirrors are now rectangular unless they are *magic*.
/:cl:

### CODE REVIEW

**Comment by nemvar** [comment]
Nice, but I'm a bit concerned about the look of the pride ruin. Can you get a picture of it if it's not too much effort?

**Comment by knacker48** [comment]
But I like round mirrors

**Comment by nicbn** [comment]
image

**Comment by kriskog** [comment]
Why is only the sink getting title and changelog time when you are also changing mirrors?

**Comment by nicbn** [comment]
Fixed

**Comment by MMMiracles** [comment]
sinks are cool but literally the mirror is just the current one but square

File: code/game/objects/structures/mirror.dm [hunk 1]
```diff
@@ -46,34 +40,31 @@
...
+		playsound(src, "shatter", 70, 1)
 		if(desc == initial(desc))
 			desc = "Oh no, seven years of bad luck!"
 		broken = TRUE
 
 /obj/structure/mirror/deconstruct(disassembled = TRUE)
 	if(!(flags_1 & NODECONSTRUCT_1))
 		if(!disassembled)
-			new /obj/item/shard( src.loc )
+			new /obj/item/shard(src.loc)
```
Comment (ShizCalev) [inline code/game/objects/structures/mirror.dm]: remove src

File: code/game/objects/structures/mirror.dm [hunk 2]
```diff
@@ -46,34 +40,31 @@
...
 		return FALSE
-
-	if(!broken)
-		return TRUE
-
-	if(!I.tool_start_check(user, amount=0))
+	if(!broken || !I.tool_start_check(user, amount=0))
 		return TRUE
 
 	to_chat(user, "<span class='notice'>You begin repairing [src]...</span>")
```
Comment (ShizCalev) [inline code/game/objects/structures/mirror.dm]: this won't work for `/obj/structure/mirror/broken`

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/mirror.dm (hunk 2 above)
Comment (nicbn) [reply code/game/objects/structures/mirror.dm]: Fixed, in a kinda-hacky-way

**Comment by HomocidlePancakes** [comment]
Why not have both kinds of windows? Plebians in the public restrooms get rectangular ones and the patricians get round ones in their private quarters.

**Comment by Krysonism** [comment]
Can we please have both styles available?

**Comment by Detective-Google** [comment]
both styles would be good.

**Review by ShizCalev — COMMENTED** [review]
image

may wanna hit these all.

**Comment by nicbn** [comment]
I'll undo the mirror changes and PR the sink sprites only

**Pull request closed without merging**